net-misc/dleyna-server/dleyna-server-0.5.0.ebuild

```diff
@@ -29,6 +29,9 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
 "
 
+# Bug 597952
+PATCHES=( "${FILESDIR}"/${P}-fix-references-to-GUPnPContextManager.patch )
+
 src_install() {
 	default
 	prune_libtool_files
```

Comment:
Check that the `.la` files can be deleted and then replace this with the find command from https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives . Then you can also nix the `inherit eutils`